File: Python/1160 Crescimento Populacional.py
n = int(input())
i = 0
while n:
    a, b, pa, pb = map(float, input().split())
    while a <= b:
        if i > 100: break
        a += a*pa//100
        b += b*pb//100
        i += 1
    if i > 100: print("Mais de 1 seculo.")
    else: print(i, "anos.")
    i = 0
    n-=1

File: C++/1160 Crescimento Populacional.cpp
#include <iostream>
using namespace std;
 
int main()
{
    int n, x = 0; cin >> n; long a, b; float pa, pb; 
    for(int i = 1; i <= n; i++){
        cin >> a >> b >> pa >> pb;
        while(a <= b){
            if(x > 100) break;
            a += a*pa/100;
            b += b*pb/100;
            x++; 
        }
        // cout << "a: " << a << endl << "b: " << b << endl << "x: " << x << endl;
        if(x > 100) cout << "Mais de 1 seculo.\n";
        else cout << x << " anos.\n";
        x = 0;
    }
 
    return 0;
}
